app/constants/chartTypeConfigs/scatterChart.js

```diff
@@ -11,3 +17,24 @@ export const config = {
 export const defaultOpts = {
   showLegend: true,
 };
+
+/**
+ * Defines size of points for scatter chart
+ */
+const scatterChartPointRange = [60, 60];
+
+/**
+ * Similar to default opts but dependent on state
+ * called by conditionalChartOptions middleware
+ * @param {Object} state Redux application state
+ *   {Object} transformedData State deconstructed
```

Comment:
Don't you want an `@param` here?

Since you only rely on a particular field in the state object, it doesn't matter if the actual state object is passed in. You could duck type it with something like `getConditionalOpts({ transformedData })`.

In that case the doc block would be:
```
 * @param {Object} transformedData transformedData State destructured
```

I think the technical term is "destructured".